Core/Utilities/DErrno.py

```diff
@@ -103,6 +103,7 @@
 
 # config
 ESECTION = 1400
+ENOVALUE = 1401
```

Comment:
Why not using the existing and standard `errno.ENODATA` ?

Reply:
fixed